### ISSUE TITLE
upgrade OpenRefine from 2.8 to 3.x

### ISSUE THREAD

**Issue by felixlohmeier** [subhh/HOS-MetadataTransformations#51]
OpenRefine 3.1 was released on Nov 29, 2018 and has some nice new features comparing to 2.8:
* release notes v3.1: https://github.com/OpenRefine/OpenRefine/releases/tag/3.1
* release notes v3.0 RC1: https://github.com/OpenRefine/OpenRefine/releases/tag/3.0-rc.1

Some functions behave differently in OpenRefine 3.x. This is a major update with breaking changes.

If we want to upgrade, we need to check all rules (e.g. by comparing tsv export).

First attempt to run everything unchanged with OpenRefine 3.1 shows some errors:
````
copernicus.sh: 36870 records 00:02:21 (hh:mm:ss) 2343 MB of 2048M
ediss.sh: 7573 records 00:01:22 (hh:mm:ss) 1284 MB of 2048M
haw.sh: java.lang.Exception: No column named translatedTitle java.lang.Exception: No column named title 3 Konfiguration scheint fehlerhaft zu sein! Bitte manuell prüfen.
java.lang.Exception: No column named title 2
java.lang.Exception: No column named translatedTitle
java.lang.Exception: No column named title 3
java.lang.Exception: No column named title 2
java.lang.Exception: No column named translatedTitle
java.lang.Exception: No column named title 3
Konfiguration für /home/etl/3-1/HOS-MetadataTransformations/bin/haw.sh scheint fehlerhaft zu sein! Bitte manuell prüfen.
````

**Comment by felixlohmeier** [comment]
There was a breaking change in OpenRefine 3.1 introduced with https://github.com/OpenRefine/OpenRefine/issues/1662 that was reverted in OpenRefine 3.2 beta with https://github.com/OpenRefine/OpenRefine/pull/1969

So we should upgrade to 3.2 beta now.

**Comment by felixlohmeier** [comment]
OpenRefine 3.2 beta seems to run fine without errors:

OpenRefine 2.8:
```
print stats and exceptions from logs...
copernicus.sh: 36870 records 00:01:37 (hh:mm:ss) 1723 MB of 2048M
ediss.sh: 7650 records 00:01:09 (hh:mm:ss) 1095 MB of 2048M
haw.sh: 4109 records 00:00:37 (hh:mm:ss) 870 MB of 2048M
hcu.sh: 453 records 00:00:19 (hh:mm:ss) 645 MB of 2048M
hsu.sh: 687 records 00:00:21 (hh:mm:ss) 750 MB of 2048M
tubdok.sh: 3217 records 00:00:33 (hh:mm:ss) 875 MB of 2048M
tuhh-fdm.sh: 34 records 00:00:16 (hh:mm:ss) 610 MB of 2048M
uhh-fis.sh: 28814 records 00:01:20 (hh:mm:ss) 1892 MB of 4096M
uke-fis.sh: 36933 records 00:03:25 (hh:mm:ss) 2712 MB of 4096M
```

OpenRefine 3.2 beta:
```
print stats and exceptions from logs...
copernicus.sh: 36870 records 00:01:35 (hh:mm:ss) 1685 MB of 2048M
ediss.sh: 7650 records 00:01:13 (hh:mm:ss) 1178 MB of 2048M
haw.sh: 4109 records 00:00:43 (hh:mm:ss) 880 MB of 2048M
hcu.sh: 453 records 00:00:24 (hh:mm:ss) 782 MB of 2048M
hsu.sh: 687 records 00:00:27 (hh:mm:ss) 792 MB of 2048M
tubdok.sh: 3217 records 00:00:39 (hh:mm:ss) 942 MB of 2048M
tuhh-fdm.sh: 34 records 00:00:21 (hh:mm:ss) 661 MB of 2048M
uhh-fis.sh: 28814 records 00:01:29 (hh:mm:ss) 2060 MB of 4096M
uke-fis.sh: 36933 records 00:03:29 (hh:mm:ss) 2994 MB of 4096M
```

**Comment by felixlohmeier** [comment]
OpenRefine 3.2 beta prints some unimportant exceptions to the console output. We need a workaround for this to make the tests in the scripts work again. I am going to change the tests in all scripts slightly with `grep -v "workspace"`. Maybe OpenRefine core developers will fix it in the final release of 3.2, see https://github.com/OpenRefine/OpenRefine/issues/1989
